frontend/src/pages/org/dashboard.ts

```diff
@@ -255,6 +255,9 @@ export class Dashboard extends BtrixElement {
                     name: "gear-wide-connected",
                     color: this.colors.crawls,
                   },
+                  button: {
+                    url: `/orgs/${this.orgId}/items/crawls`,
```

Comment:
Using `this.navigate.orgBasePath` will remove the extra processing of an org ID to slug. Also, `/crawls` is not a valid path, it should be `crawl`--same for `upload`.
```suggestion
                    url: `/${this.navigate.orgBasePath}/items/crawl`,
```

Reply:
My bad, thank you!